internal/testdata/channels.go

```diff
@@ -49,3 +49,150 @@ func mockConsensusChannel(counterparty types.Address) (ledger *consensus_channel
 	cc, _ := testObj.CreateConsensusChannel()
 	return cc, true
 }
+
+// LedgerNetwork is a collection of in-memory consensus_channel ledgers
+// which expose both the leader and follower perspective on the ledger
+type LedgerNetwork struct {
+	ledgers []TestLedger
+}
+
+// TestLedger wraps the leader and follower views of a consensus_channel
+type TestLedger struct {
+	LeaderView   consensus_channel.ConsensusChannel
+	FollowerView consensus_channel.ConsensusChannel
+}
+
+// GetLedgerLookup returns a ledger-lookup function for the given ledger seeker.
+//
+// The returned function inspects the ledgers from the ledger set, and returns
+// the ledger between the seeker and given counterparty from the seeker's perspective.
+func (l LedgerNetwork) GetLedgerLookup(seeker types.Address) virtualfund.GetTwoPartyConsensusLedgerFunction {
+	var myLedgers []TestLedger
+
+	// package all of seeker's ledgers for the closure
+	for _, ledger := range l.ledgers {
+		if ledger.FollowerView.Leader() == seeker ||
+			ledger.FollowerView.Follower() == seeker {
+			myLedgers = append(myLedgers, ledger)
+		}
+	}
+
+	return func(counterparty types.Address) (ledger *consensus_channel.ConsensusChannel, ok bool) {
+		for _, ledger := range myLedgers {
+			if ledger.FollowerView.Follower() == seeker &&
+				ledger.FollowerView.Leader() == counterparty {
+				return &ledger.FollowerView, true
+			}
+
+			if ledger.LeaderView.Leader() == seeker &&
+				ledger.LeaderView.Follower() == counterparty {
+				return &ledger.LeaderView, true
+			}
+		}
+		return nil, false
+	}
+}
+
+// createLedgerNetwork returns active, funded consensus_channels connecting the supplied
```

Comment:
Super nit: what does "active" mean?

Reply:
approximately:
- has a signed post-fund state
- is ready to `Propose`, `SignNextProposal`, etc